pkg/session_recording/recordings.jsx

```diff
@@ -395,16 +397,13 @@
 
             for (i = 0; i < entryList.length; i++) {
                 let e = entryList[i];
-                let boot_id = e["_BOOT_ID"];
-                let session_id = e["TLOG_SESSION"];
-                let process_id = e["_PID"];
+                let id = e['TLOG_REC'];
 
                 /* Skip entries with missing session ID */
-                if (session_id === undefined) {
+                if (id === undefined) {
                     continue;
                 }
```

Comment:
Thanks, Kyrill! However, you seem to have omitted the change from your recent push, as the diff still shows the old comment.